astroquery/alma/__init__.py

```diff
@@ -19,10 +27,49 @@ class Conf(_config.ConfigNamespace):
     Configuration parameters for `astroquery.alma`.
     """
 
+    # Used if the current discovered host has no Service ID equivalent.  This
+    # is used when overriding the default registry, which assumes to have
+    # just almascience.org as the authority in the Service IDs.
+    default_service_id_auth = 'almascience.org'
+
+    service_uri_map = _arc_service_id_map
+
+    registry_path = _config.ConfigItem(
+        '/reg/resource-caps',
+        'ALMA Registry path')
+
+    tap_service_uri_path = _config.ConfigItem(
+        '/tap',
+        'ALMA TAP ObsCore service URI path')
+
+    tap_standard_id = _config.ConfigItem(
+        'ivo://ivoa.net/std/TAP',
+        'ALMA TAP service standard ID')
+
+    datalink_service_uri_path = _config.ConfigItem(
+        '/datalink',
+        'ALMA DataLink service URI path')
+
+    datalink_standard_id = _config.ConfigItem(
+        'ivo://ivoa.net/std/DataLink#links-1.0',
+        'ALMA DataLink service standard ID'
+    )
+
+    sia_service_uri_path = _config.ConfigItem(
+        '/sia',
+        'ALMA SIAv2 service URI path')
+
+    sia_standard_id = _config.ConfigItem(
+        'ivo://ivoa.net/std/SIA#query-2.0',
+        'ALMA SIAv2 service standard ID')
+
     timeout = _config.ConfigItem(60, "Timeout in seconds.")
 
+    # override for testing or alternate archive location.
+    _configured_archive_url = [os.environ["ARCHIVE_URL"]] if "ARCHIVE_URL" in os.environ else _url_list
```

Comment:
Why is an environment variable required at all? If testing requires a different value then why wouldn't it be possible to change the configuration variable during test setup instead?

Reply:
It's not needed.  I was thinking of a runtime way of setting it, but this will be used in scripting and the `archive_url` can simply be set that way.  This will be removed.